components/patient-access-rules/api/src/test/java/org/phenotips/data/permissions/internal/DefaultPatientAccessHelperTest.java

```diff
@@ -316,9 +316,9 @@ public void getCollaborators() throws Exception
         when(manager.resolveAccessLevel("view")).thenReturn(view);
         Collection<Collaborator> collaborators = this.mocker.getComponentUnderTest().getCollaborators(this.patient);
         Assert.assertEquals(2, collaborators.size());
-        Collaborator c = new DefaultCollaborator(COLLABORATOR, edit, this.mocker.getComponentUnderTest());
```

Comment:
Looks like these test no longer depend on the component under test (PatientAccessHelper) - so it seems these tests should be removed from this test file